newsletter-march-2023.rst

```diff
@@ -0,0 +1,127 @@
+  we will start to roll it out as a kind of A/B testing.
+- ⏳️ The first work from our Diátaxis documentation refactor is now live and the general work has moved into shorter iterations.
+  Once we have gathered all the final experience from the refactor,
+  we will announce it together with practical inputs for how other documentation projects might approach such a refactor.
+  If you are planning a refactor of a large documentation,
+  we'd love to hear what your questions and concerns are and address those future blog posts and talks.
+- 🎬️ We're slowly moving towards a public beta of the whole Read the Docs dashboard experience. Here's a screenshot of what's to come...
+
+.. image:: img/screenshot-dashboard-beta.png
+   :align: center
+   :scale: 50%
+
+
+Interested in all the details? `View our full Roadmap 📍️`_
+
+.. _View our full Roadmap 📍️: https://github.com/orgs/readthedocs/projects/156/views/1
+
+Possible issues
+---------------
+
+- TBD
+
+
+Awesome Project of the month
+----------------------------
+
+`The Haskell Tool Stack <https://docs.haskellstack.org/>`__ is commonly known in the world of `Haskell <https://en.wikipedia.org/wiki/Haskell>`__ as simply Stack and is this month's chosen entry from `Awesome Read the Docs Projects 🕶️ <https://github.com/readthedocs-examples/awesome-read-the-docs>`_.
+See our chosen highlights from Stack's documentation in the following `Twitter thread <https://twitter.com/readthedocs/status/1603095976117522433>`_:
+
+.. raw:: html
+
+   <blockquote class="twitter-tweet"><p lang="en" dir="ltr">GeoPandas is an open source project to make working with <a href="https://twitter.com/hashtag/geospatial?src=hash&amp;ref_src=twsrc%5Etfw">#geospatial</a> data in <a href="https://twitter.com/hashtag/Python?src=hash&amp;ref_src=twsrc%5Etfw">#Python</a> easier. <a href="https://twitter.com/geopandas?ref_src=twsrc%5Etfw">@GeoPandas</a> extends the datatypes used by pandas to allow spatial operations on geometric types.<br><br>We want to highlight some things we love from their docs.<br><br>🤏 (small) 🧵 <a href="https://t.co/Hj82s6SDQP">pic.twitter.com/Hj82s6SDQP</a></p>&mdash; Read the Docs (@readthedocs) <a href="https://twitter.com/readthedocs/status/1603095976117522433?ref_src=twsrc%5Etfw">December 14, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+Tip of the month
+----------------
+
+When you post links to your documentation on chat and social media,
+you will probably see a very uninspiring and generic preview.
+Enter `sphinxext-opengraph <https://github.com/wpilibsuite/sphinxext-opengraph>`__!
+
+The extension allows you to configure your own preview card,
+compatible with all major chat and social media platforms.
+The feature which we really enjoy is that it will detect and use the first illustration used in your documentation.
+If you want to see it in action,
+try sharing a link to this newsletter blog post (marketing unintended).
```

Comment:
We should be promoting this more. Not sure the best avenue, but it's a great extension. 

We should probably note our official docs are using it above in the new features ^^

Reply:
Added a link to the user docs as well!